src/transformers/hyperparameter_search.py

```diff
@@ -44,7 +44,7 @@ class HyperParamSearchBackendBase:
     def is_available():
         raise NotImplementedError
 
-    def run(self, trainer, n_trials: int, direction: str, **kwargs):
+    def run(self, trainer, n_trials: int, direction: str, use_best_model: bool, **kwargs):
```

Comment:
There should be a default value for `use_best_model`, otherwise would be a breaking change.